src/codewhisperer/commands/basicCommands.ts

```diff
@@ -69,6 +69,19 @@ export const enableCodeSuggestions = Commands.declare(
     }
 )
 
+export const disableAutoTriggerOfCodeSuggestions = Commands.declare(
+    'aws.codeWhisperer.disableAutoTriggerOfCodeSuggestions',
```

Comment:
unnecessarily verbose. there is no other "auto trigger" that needs to be disambiguated.
```suggestion
    'aws.codeWhisperer.disableAutoTrigger',
```